packages/jest-message-util/src/__tests__/messages.test.ts

```diff
@@ -289,3 +289,79 @@ it('no stack', () => {
 
   expect(message).toMatchSnapshot();
 });
+
+describe('getConsoleOutput', () => {
+  it('prints code frame and stacktrace', () => {
+    readFileSync.mockImplementationOnce(() => 'throw new Error("Whoops!");');
+    const message = formatStackTrace(
+      `
+      at Object.<anonymous> (${slash(rootDir)}/file.js:1:7)
+      at Module._compile (internal/modules/cjs/loader.js:1158:30)
+      at Object.Module._extensions..js (internal/modules/cjs/loader.js:1178:10)
+      at Module.load (internal/modules/cjs/loader.js:1002:32)
+      at Function.Module._load (internal/modules/cjs/loader.js:901:14)
+      at Function.executeUserEntryPoint [as runMain] (internal/modules/run_main.js:74:12)
+  `,
+      {
+        rootDir,
+        testMatch: [],
+      },
+      {
+        noCodeFrame: false,
+        noStackTrace: false,
+      },
+      'path_test',
+    );
+
+    expect(message).toMatchSnapshot();
```

Comment:
```suggestion
    expect(wrap(message)).toMatchSnapshot();
```

`import {wrap} from 'jest-snapshot-serializer-raw';`

will get rid of the wrapping "`.

Please apply to both 😀